backend/src/test/java/com/example/backend/controller/TripsControllerTest.java

```diff
@@ -171,4 +171,62 @@ void deleteById_Throws404WhenIdNotRegistered() throws Exception {
         mvc.perform(delete("/api/trips/NONEREGISTEREDID"))
                 .andExpect(status().isNotFound());
     }
+
+    @Test
+    public void update_updatesTripIfExists() throws Exception {
+        // given
+        Trip existingTrip = testTrip1();
+        this.tripRepo.save(existingTrip);
+
+        String json = String.format("""
```

Comment:
Just calling it json feels a bit ambiguous to me?

Reply:
Hast du nen Vorschlag? das is ja quasi das json, was ich reingebe und "nicht strict" auch wieder zurück erwarte. Wußte nich genau, wie ich es sonst nennen sollte

Reply:
done